superset/migrations/versions/2024-05-24_11-31_02f4f7811799_remove_sl__tables.py

```diff
@@ -24,21 +24,41 @@
 
 import sqlalchemy as sa
 from alembic import op
+from sqlalchemy.engine.reflection import Inspector
 
 # revision identifiers, used by Alembic.
 revision = "02f4f7811799"
 down_revision = "f7b6750b67e8"
 
+tables_to_drop = [
+    "sl_dataset_columns",
+    "sl_table_columns",
+    "sl_dataset_tables",
+    "sl_columns",
+    "sl_tables",
+    "sl_dataset_users",
+    "sl_datasets"
+]
+
+def drop_constraints(table, insp):
```

Comment:
This function seems useful and could be moved into `superset/migrations/shared/utils.py` so it can be reused in future migrations. I'd make both `insp` and `conn` arguments so the caller can reuse theirs and control transations.